schema/preset/v2.json

```
@@ -93,17 +93,23 @@
       "items": {
         "type": "string"
       }
+    },
+    "color": {
+      "description": "string representation of a color in 24 bit (#rrggbb)",
+      "type": "string",
+      "pattern": "^#([a-fA-F0-9]{6})$"
```

Nitpick: I don't think we need this [capturing group](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Capturing_group).

```suggestion
      "pattern": "^#[a-fA-F0-9]{6}$"
```

---

Yeah, it was left from when it also mached against 12-bit color (so, like #rgb)